OSDOCS CQA AUTH-4 Service Accounts and Cloud Credential Operator I (#115126)
openshift/openshift-docs: authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc

:_mod-docs-content-type: ASSEMBLY
[id="cco-mode-mint"]
= About the Cloud Credential Operator in mint mode
include::_attributes/common-attributes.adoc[]
:context: cco-mode-mint

toc::[]

[role="_abstract"]

> You can use the Cloud Credential Operator (CCO) in mint mode to create new credentials for components in the cluster with only the specific permissions that are required.

Reviewer: I would make sure it's clear that this is saying that the **components** only have the perms they need. The cluster itself has an admin cred stored in the `kube-system` namespace in this mode. One of the key benefits of moving to the manual + token-based auth strategy is to get the admin cred out of the cluster. 

The conceptual info you modularized goes into this, so maybe it would be best to leave that part out up here and focus on the basic benefits:

```suggestion
You can use the Cloud Credential Operator (CCO) in mint mode to create and reconcile credentials for components in the cluster.
```

> Mint mode is the default Cloud Credential Operator (CCO) credentials mode for {product-title} on platforms that support it. Mint mode supports {aws-first} and {gcp-first} clusters.

Reviewer: Defined in the para right above

```suggestion
Mint mode is the default CCO credentials mode for {product-title} on platforms that support it. Mint mode supports {aws-first} and {gcp-first} clusters.
```

:_mod-docs-content-type: ASSEMBLY
[id="cco-mode-mint"]
= The Cloud Credential Operator in mint mode
= Understanding the Cloud Credential Operator in mint mode
include::_attributes/common-attributes.adoc[]
:context: cco-mode-mint

toc::[]

Mint mode is the default Cloud Credential Operator (CCO) credentials mode for {product-title} on platforms that support it. Mint mode supports Amazon Web Services (AWS) and {gcp-first} clusters.
[role="_abstract"]
You can use the Cloud Credential Operator (CCO) in mint mode to create new credentials for components in the cluster with only the specific permissions that are required.

[id="mint-mode-about"]

> == Mint mode credentials management

Author: The removed text moved to new mint-mode-credentials-management and mint-mode-permission-requirements modules.
